index.html

```diff
@@ -1,21 +1,270 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="index.css" />
-    <title>Credard</title>
-  </head>
-  <body>
-    <!-- where the nav should be -->
-    <header class="landing-page_header">
-      <h1>
-        Say bye to boring credit cards, get your own
-        <span>exciting designs!</span>
-      </h1>
-    </header>
-    <main></main>
-    <footer></footer>
-  </body>
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="index.css" />
+        <title>Credard</title>
+    </head>
+    <body>
+        <!-- where the nav should be -->
+        <main>
+            <section class="hero">
+                <figure class="hero_fig">
+                    <div class="hero_img-c">
+                        <img
+                            class="hero_img"
+                            src="./assets/images/credit-cards-hero.svg"
+                            alt=""
+                        />
+                    </div>
+                    <figcaption class="hero_figc">
+                        <h1>
+                            Say bye to boring credit cards, get your own
+                            <span>exciting designs!</span>
+                        </h1>
+                        <div class="hero_links">
+                            <a class="hero_link hero_link-1" href="#"
+                                >View our library</a
+                            >
+                            <a class="hero_link hero_link-2" href="#"
+                                >How does it work?</a
+                            >
+                        </div>
+                    </figcaption>
+                </figure>
+            </section>
+            <section class="features">
+                <figure class="feature">
+                    <div class="feature_img-c">
+                        <img
+                            class="feature_img"
+                            src="./assets/images/credit-card-payment-cuate.svg"
+                            alt=""
+                        />
+                    </div>
+                    <figcaption class="feature_figc">
+                        <h3 class="feature_title">
+                            Upgrade your boring credit card
+                        </h3>
+                        <p class="feature_text">
+                            Pick a design from our library and print it out as
+                            sticker skin for your credit card.
+                        </p>
+                        <button class="feature_btn">View our library</button>
```

Comment:
Add a href to the button and cursor pointer
![image](https://user-images.githubusercontent.com/60205816/183245420-740db7ac-30f5-4108-982f-9ff87ba5777a.png)